src/i18n/locales/en/translation.json

```
@@ -145,8 +149,50 @@
     "placeholder_seed_word_input": "Word",
     "hint_duration_text": "Please be patient, this may take a few minutes."
   },
+  "import_wallet": {
+    "alert_other_wallet_unlocked": "Currently <1>{{ walletName }}</1> is active. You need to lock it first. <3>Go back</3>.",
+    "alert_rescanning_already_in_progress": "Rescanning the timechain is currently in progress. Please wait until the process finishes and then try again. <1>Go back</1>.",
+    "wallet_details": {
+      "title": "Import Wallet",
+      "text_button_submit": "Continue",
+      "text_button_submitting": "Continue"
+    },
+    "import_details": {
+      "title": "Please enter your mnemonic phrase!",
+      "subtitle": "Please enter your mnemonic phrase one by one in the correct order and start the rescan process.",
+      "import_options": "Import options",
+      "label_blockheight": "Rescan height",
+      "description_blockheight": "The blockheight at which the rescan process starts to search for your funds. The earlier the wallet has been created, the lower this value should be. ",
+      "feedback_invalid_blockheight": "Please provide a valid blockheight value greater than or equal to {{ min }}.",
+      "label_gaplimit": "Gaplimit",
+      "description_gaplimit": "The number of unused addresses encountered beyond which no further addresses are imported. Increase this number if you created a lot of addresses which never received any funds.",
```

Despite the usual definition the gaplimit setting for JoinMarket needs to be the highest number of address used in any mixdepth.

Attempting to improve the wording in the Jam context:
```suggestion
      "description_gaplimit": "Set to the highest address number used in any of the jars. Increase this number if your wallet is heavily used.",
```

---

Agree. Let's not try to redefine the gaplimit used in other software.
How about "address height"?